tools/nnyeah/nnyeah/Program.cs

```diff
@@ -58,15 +58,19 @@ static void Main (string [] args)
 			bool forceOverwrite, bool suppressWarnings)
 		{
 			if (!TryLoadTypeAndModuleMap (xamarinAssembly!, microsoftAssembly!, publicOnly: true,
-				out var typeAndModuleMap, out var failureReason)) {
+				out var typeAndModuleMap, out var failureReason, out var xamarinModule,
+				out var microsoftModule)) {
 				Console.Error.WriteLine (Errors.E0011, failureReason);
 			}
-			ReworkFile (infile!, outfile!, verbose, forceOverwrite, suppressWarnings, typeAndModuleMap!);
+			ReworkFile (infile!, outfile!, verbose, forceOverwrite, suppressWarnings, typeAndModuleMap!,
+				xamarinModule!, microsoftModule!);
```

Comment:
Inverting the if will remove all those !
```csharp
if (TryLoadTypeAndModuleMap (xamarinAssembly!, microsoftAssembly!, publicOnly: true,
 				out var typeAndModuleMap, out var failureReason)) {
    ReworkFile (infile!, outfile!, verbose, forceOverwrite, suppressWarnings, typeAndModuleMap,
  				xamarinModule, microsoftModule);
} else {
    Console.Error.WriteLine (Errors.E0011, failureReason);
}
```

Give it a go, but not a show stopper.